pages/docs/setup/connect.mdx

```diff
@@ -367,6 +367,14 @@ This view is helpful for debugging connection issues or verifying rolling deploy
 
 ![App worker observability](/assets/docs/connect/cloud-app-workers.png)
 
+## Syncing and Rollbacks
+
+When deploying a new version of your application, the first instance to reach `connect()` will kick off a sync during the connection setup process. This will update function configurations to the desired state configured in your code.
```

Comment:
```suggestion
When you deploy a new version of your application, the first worker to connect to Inngest will automatically sync your app. This will update function configurations to the desired state configured in your code.
```

Using some terms from this and other docs: "sync your app" and "worker" - we could also link the lifecycle section which mentions this as well.